meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -662,7 +662,7 @@ UBOOT_LOADADDRESS:am62xx = "0x82000000"
 UBOOT_DTB_LOADADDRESS:am62xx = "0x88000000"
 UBOOT_DTBO_LOADADDRESS:am62xx = "0x89000000"
 EFI_PROVIDER:am62xx = ""
-LMP_BOOT_FIRMWARE_FILES:am62xx = "tispl.bin u-boot.img tiboot3.bin"
+LMP_BOOT_FIRMWARE_FILES:am62xx = "tispl.bin u-boot.img tiboot3-am62x-gp-evm.bin tiboot3-am62x-hs-evm.bin tiboot3-am62x-hs-fs-evm.bin"
```

Comment:
Unfortunately it's not possible to rename file using `;`, as it's done for `IMAGE_BOOT_FILES`:
```
IMAGE_BOOT_FILES:append:am62xx-evm = " tiboot3-am62x-gp-evm.bin;tiboot3.bin"
```
So this is why I copy all three for now.

Reply:
Yeah, this is fine as the usage here is for the boot firmware files recipe anyway.